lib/components/narrative/narrative-itineraries.js

```diff
@@ -12,7 +12,7 @@ import {
   setVisibleItinerary,
   updateItineraryFilter
 } from '../../actions/narrative'
-import { ItineraryView } from '../../actions/ui'
+import { ItineraryView, setItineraryView } from '../../actions/ui'
```

Comment:
I suppose this file isn't already doing this, but I do prefer these action imports to follow the format used elsewhere, i.e.:

```
import * as uiActions from '../../actions/ui'
```

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/48bafb663209bf2e84d22b0c491709cba7c31919.